tests/shardddl3/run.sh

```diff
@@ -802,6 +802,61 @@ function DM_RestartMaster() {
 }
 
 function DM_SyncView_CASE() {
+    # test sharding lock mixed VIEW
+    # 1/3 shard DDL
+    run_sql_source1 "alter table ${shardddl1}.${tb1} add column c int;"
+
+    run_sql_source1 "create view ${shardddl1}.v1 as select * from ${shardddl1}.${tb1};"
+    sleep 1
+    run_sql_tidb "show create view ${shardddl}.v"
+    check_contains "View: v"
```

Comment:
Seems view is created here in pessimistic?

Reply:
yes. VIEW is pretend to sync in none ShardMode, try to immediately execute to downstream.

Reply:
But the table may not be created?
```
create table                        <-- not sync
create view                         <-- sync
                     create table
                     create view
```